<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:black;
}
QWidget{
background-color:black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
background-color:black;
}
QTabBar::tab{
color:white;
background-color:black;
width:50px;
height:105px;
}
QTabBar{
color:black;
background-color:black;
}
QWidget{
background-color:black;
border-color:black;
}
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Battery</string>
     </attribute>
     <widget class="QProgressBar" name="current_regen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(45, 137, 25);
border-radius:0px;
}
QProgressBar{
border: 2px solid grey;
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Regen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="SOC_prog">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>31</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar{
border: 2px solid black;
border-radius: 5px;
text-align: center;
background:white;
}
QProgressBar::chunk{
border-radius: 5px;
}

</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="temp_prog">
      <property name="geometry">
       <rect>
        <x>345</x>
        <y>20</y>
        <width>51</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color: rgb(188, 5, 5);
border-radius: 3px;
}
QProgressBar{
border: none;
border-radius: 10px;
text-align: center;
background:black;
color:white;}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="current_discharge">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(251, 90, 11);
border-radius:0px;
}
QProgressBar{
border: 2px solid grey;
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Discharge&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="bat_out">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>28</y>
        <width>299</width>
        <height>97</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/bo.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>101</width>
        <height>295</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/thermooutline.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>180</y>
        <width>13</width>
        <height>81</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="regen_num">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>64</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
text-align:center;
}</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="dischargenum">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>bat_out</zorder>
     <zorder>SOC_prog</zorder>
     <zorder>current_regen</zorder>
     <zorder>label_2</zorder>
     <zorder>temp_prog</zorder>
     <zorder>current_discharge</zorder>
     <zorder>label</zorder>
     <zorder>line</zorder>
     <zorder>regen_num</zorder>
     <zorder>dischargenum</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Power</string>
     </attribute>
     <widget class="QProgressBar" name="EnginePow">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar{
border: 2px solid black;
border-radius: 5px;
text-align: center;
color:white;
background:grey;
}
QProgressBar::chunk{
border-radius: 5px;
color: red;
background-color:rgb(149, 11, 11);
}
</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="MotPow">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar{
border: 2px solid black;
border-radius: 5px;
text-align: center;
color:white;
background:grey;
}
QProgressBar::chunk{
border-radius: 5px;
color: red;
background-color:rgb(25, 143, 41);
}
</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Engine Power</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Motor  Power</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Flow</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>421</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/camaro.bmp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fatboy">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>126</y>
        <width>151</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/arrowleft.bmp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/chub.bmp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="skinnyboy">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/arrowleft.bmp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="dischargearrow">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/unchrg.bmp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="chargarrow">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="application.qrc">:/chrg.bmp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>88</x>
        <y>82</y>
        <width>5</width>
        <height>135</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(234, 170, 12);
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="orb1_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk{
background-color:rgb(252, 149, 0);
border-radius:5px;
}
QProgressBar{
border: 2px solid rgb(252, 149, 0);
border-radius: 10px;
text-align: center;
background:black;
color:white;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
